R/pkg/R/types.R

```diff
@@ -67,3 +67,24 @@ rToSQLTypes <- as.environment(list(
   "double" = "double",
   "character" = "string",
   "logical" = "boolean"))
+
+# Helper function of coverting decimal type. When backend returns column type in the
+# format of decimal(,) (e.g., decimal(10, 0)), this function coverts the column type
+# as double type.
```

Comment:
could you add `@param`? no need to prefix with `#'` since this is not going to Rd/documentation.